// @test error

// Attempt to return value from function with no return type

fn foo()
{
	return 4;
}

fn main() : int
{
	return 4;
}